db/sessions.go

```diff
@@ -32,6 +32,12 @@ func QuerySessions() []Session {
 	return ExecuteSessionsQuery(query)
 }
 
+// Deletes the sessions and messages of the given user
+func DeleteSessionsWithMessages(username string) bool {
```

Comment:
To make things "safer" we should get in the habit of making these functions private if they are really going to be private. So you should make this lowercase.